_getting-started/intro.md

```diff
@@ -0,0 +1,161 @@
+
+An inverted index for such an index maps the words to the documents in which they occur:
+
+Word | Document
+:--- | :---
+beauty | 1, 2
+is | 1
+in | 1
+the | 1, 2
+eye | 1
+of | 1
+the | 1
+beholder | 1
+and | 2
+beast | 2 
+
+In addition to the document ID, OpenSearch stores the position of the word within the document for running phrase queries, where words must appear next to each other.
+
+## Relevance
+
+When you search for a document, OpenSearch matches the words in the query to the words in the documents. For example, if you search the index described in the previous section for the word `beauty`, OpenSearch will return documents 1 and 2. Each document is assigned a _relevance score_ that tells you how well the document matched the query.
+
+Individual words in a search query are called search _terms_. Each search term is scored according to the following rules:
+
+1. A search term that occurs more frequently in a document will tend to be scored higher. A document about dogs that uses the word `dog` many times is likely more relevant than a document that contains the word `dog` fewer times. This is the _term frequency_ component of the score.
+
+1. A search term that occurs in more documents will tend to be scored lower. A query for the terms `blue` and `axolotl` should prefer documents that contain `axolotl` over the likely more common word `blue`. This is the _inverse document frequency_ component of the score.
+
+1. A match on a longer document should tend to be scored lower than a match on a shorter document. A document that contains a full dictionary would match on any word but is not very relevant to any particular word. This corresponds to the _length normalization_ component of the score.
+
+OpenSearch uses the BM25 ranking algorithm to calculate document relevance scores and then returns the results sorted by relevance. To learn more, see [Okapi BM25](https://en.wikipedia.org/wiki/Okapi_BM25).
+
+## Advanced concepts
+
+The following section describes more advanced OpenSearch concepts.
+
+### Update lifecycle
+
+The lifecycle of an update operation consists of the following steps:
+
+1. An update is received by a primary shard and is written to the shard's transaction log ([translog](#translog)). The translog is flushed to disk (followed by an fsync) before the update is acknowledged. This guarantees durability.
+1. The update is also passed to the Lucene index writer, which adds it to an in-memory buffer.
+1. On a [refresh operation](#refresh), the Lucene index writer flushes the in-memory buffers to disk (with each buffer becoming a new Lucene segment), and a new index reader is opened over the resulting segment files. The updates are now visible for search.
+1. On a [flush operation](#flush), the shard fsyncs the Lucene segments. Because the segment files are a durable representation of the updates, the translog is no longer needed to provide durability, do the updates can be purged from the translog.
```

Comment:
```suggestion
1. On a [flush operation](#flush), the shard fsyncs the Lucene segments. Because the segment files are a durable representation of the updates, the translog is no longer needed to provide durability, so the updates can be purged from the translog.
```